common/types/shared.go

```diff
@@ -1710,6 +1710,13 @@ type DescribeDomainResponse struct {
 	FailoverInfo             *FailoverInfo                   `json:"failoverInfo,omitempty"`
 }
 
+func (v *DomainReplicationConfiguration) GetActiveClusters() (o *ActiveClusters) {
```

Comment:
this is not new, it's just shuffled from below due to my removing and then re-adding it. Happy to put it back in it's previous spot if necessary